pkg/connection/store/kubernetes/client.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
```

Comment:
Copied from provider-kubernetes. We also have the same in provider-helm. 
It makes sense all to use this one in runtime.

Reply:
Yeah, since this is the third copy of this and we haven't seen any divergence yet I agree we can consolidate here. :)